docs/demos/zdm/Connect_Power_Bi_to_the_Zerynth_Device_Manager.md

```diff
@@ -9,7 +9,7 @@ In this tutorial, the Power Bi Desktop is used to build a dashboard that connect
 To start with this tutorial you need:
 
 - A Zerynth Device Manager account with an active workspace where at least one device has been created,
-- A Zerynth device running the [simple ZDM example](https://docs.zerynth.com/latest/reference/libs/zerynth/zdm/docs/examples/#simple-zdm) (see the [ZDM getting started](https://docs.zerynth.com/latest/deploy/) for how to program and connect a device to the ZDM) or a PC/Raspberry Pi running the ZDM Client Python lib example (see the dedicated how-to guide [here](/latest/deploy/)),
+- A Zerynth device running the [simple ZDM example](https://docs.zerynth.com/latest/reference/libs/zerynth/zdm/docs/examples/#simple-zdm) (see the [ZDM getting started](https://docs.zerynth.com/latest/deploy/getting_started_with_sdk/) for how to program and connect a device to the ZDM) or a PC/Raspberry Pi running the ZDM Client Python lib example (see the dedicated how-to guide [here](https://docs.zerynth.com/latest/deploy/getting_started_with_rpi/),
```

Comment:
The links to the docs should not be "hard coded" with the full path , but they should only continas the relative path.
For example,
Form this one`https://docs.zerynth.com/latest/reference/libs/zerynth/zdm/docs/examples/#simple-zdm` to `/latest/reference/libs/zerynth/zdm/docs/examples/#simple-zdm`